ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/services/reflection/ReflectionIT.scala

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.platform.sandbox.services.reflection
+
+import com.digitalasset.ledger.api.testing.utils.SuiteResourceManagementAroundAll
+import com.digitalasset.platform.sandbox.services.SandboxFixture
+import io.grpc.reflection.v1alpha.{
+  ServerReflectionGrpc,
+  ServerReflectionRequest,
+  ServerReflectionResponse
+}
+import io.grpc.stub.StreamObserver
+import org.scalatest.{AsyncWordSpec, Matchers}
+
+import scala.collection.JavaConverters._
+import scala.concurrent.{Future, Promise}
+
+final class ReflectionIT
+    extends AsyncWordSpec
+    with Matchers
+    with SandboxFixture
+    with SuiteResourceManagementAroundAll {
+
+  private val listServices: ServerReflectionRequest =
+    ServerReflectionRequest.newBuilder().setHost("127.0.0.1").setListServices("").build()
+
+  "Reflection service" when {
+
+    "accessed" should {
+
+      "provide a list of exposed services" in {
+        for {
+          response <- execRequest(listServices)
+        } yield {
+          response.getListServicesResponse.getServiceCount shouldEqual 13
```

Comment:
@SamirTalwar I'm aware this is in conflict with #3573, I'll wait until that's merged to rebase and change this.

Reply:
🖖

Reply:
Rebased.